managed/src/main/java/com/yugabyte/yw/common/AlertManager.java

```diff
@@ -87,10 +93,41 @@ public void sendEmail(Alert alert, String state) {
             customer.name, state, alert.message);
       }
     }
+    subject = String.format("Yugabyte Platform Alert ERROR- <%s> %s", customer.getTag(),
+      universe.name);
+    String style = String
+      .format("%s font-size: 14px;background-color:#f7f7f7;padding:25px 30px 5px;", STYLE_FONT);
+    String hostname = "";
+    String ip = "";
+    try {
```

Comment:
As we have this code already in several places, let's extract it into com.yugabyte.yw.common.Util (or another utility class if you feel it suits better).